scripts/iscan

```diff
@@ -44,8 +44,9 @@ Positional arguments:
   <volume>    Path to a directory, path to a block device, or ID of Elastio recovery point
 
 Environment variables:
-  S3_BUCKET   S3 bucket to upload scan results to (default: '$DEFAULT_S3_BUCKET').
-              Setting S3_BUCKET to an empty string disables S3 upload.
+  S3_BUCKET           S3 bucket to upload scan results to (default: '$DEFAULT_S3_BUCKET').
+                      Setting S3_BUCKET to an empty string disables S3 upload.
+  ISCAN_EXTRA_PARAMS  Extra options for ransomware detection for developers.
```

Comment:
Let's not pollute customer-facing CLI with our internal peculiarities.
```suggestion
```
In the future we may move these configuration parameters into `~/.config/iscan/config.yml`.  For now I think it's better to leave this option undocumented.